65.md

```diff
@@ -0,0 +1,70 @@
+NIP-65
+======
+
+Relay List Metadata
+-------------------
+
+`draft` `optional` `author:mikedilger`
+
+A special replaceable event meaning "Relay List Metadata" is defined as an event with kind `10002` having a list of `r` tags, one for each relay the author uses to either read or write to.
+
+The primary purpose of this relay list is to advertise to others, not for configuring one's client.
+
+The content is not used and SHOULD be blank.
```

Comment:
I think it's more clear to say "an empty string" here rather than "blank".

Reply:
Sure.

This is merged so I think you should do that change in another PR for quick merging (unless there is a way to do it from here I'm unaware of).